base_comment_template/models/comment_template.py

```diff
@@ -26,7 +26,10 @@ class CommentTemplate(models.AbstractModel):
         compute_sudo=True,
         comodel_name="base.comment.template",
         string="Comment Template",
-        domain=lambda self: [("model_ids", "in", self._name)],
+        domain=lambda self: [
+            ("model_ids", "in", self._name),
+            ("global_template", "=", False),
```

Comment:
What confuses me is this change... You need to remove it and leave only the other.